src/Commands/WorkflowDumpCommand.php

```diff
@@ -43,7 +45,9 @@ public function handle()
         $workflowName = $this->argument('workflow');
         $format = $this->option('format');
         $class = $this->option('class');
+        $disk = $this->option('disk');
         $config = Config::get('workflow');
+        $path = Storage::disk($disk)->getAdapter()->getPathPrefix();
```

Comment:
It would be nice if this could be a bit more flexible. This locks everyone into specifying the path as the root of a storage disk. I'm thinking it could be more useful to more people if we could default the disk to `local`, and allow an optional path within that disk.

Reply:
now it's possible to specify an optional path like you have proposed
and the new default disk now is **local**
ex :  `php artisan workflow:dump workflow_name --class App\\BlogPost --path=your/path`
the output will be in : `storage/app/your/path`